libs/core/src/true-affected.ts

```diff
@@ -23,6 +23,7 @@ export const trueAffected = async ({
   base = 'origin/main',
   projects,
   includeFiles = [],
+  testSuffixes = ['spec', 'test'],
```

Comment:
Are we sure that only tests are things that can create this problem? What about types? Or bin files?

Reply:
Let's start with these, and later improve by requirements.
Also I'd give control from the cli of its required

Reply:
what i am saying is that maybe the name of the option (`testSuffixes`) is wrong